lms/templates/courseware/course_navigation.html

```diff
@@ -25,7 +25,7 @@
 %>
 
 % if show_preview_menu:
-    <nav class="wrapper-preview-menu">
+    <nav class="wrapper-preview-menu" aria-label="${_('View course as')}">
```

Comment:
Let's change the label to "Course view"  The screen reader will display/announce it as "Course view, navigation"